2024-03-08 00:41:51,011 INFO    MainThread:9152 [wandb_setup.py:_flush():76] Current SDK version is 0.16.4
2024-03-08 00:41:51,011 INFO    MainThread:9152 [wandb_setup.py:_flush():76] Configure stats pid to 9152
2024-03-08 00:41:51,011 INFO    MainThread:9152 [wandb_setup.py:_flush():76] Loading settings from C:\Users\hp\.config\wandb\settings
2024-03-08 00:41:51,011 INFO    MainThread:9152 [wandb_setup.py:_flush():76] Loading settings from C:\Users\hp\Desktop\CV\Assignment_2\wandb\settings
2024-03-08 00:41:51,011 INFO    MainThread:9152 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-03-08 00:41:51,011 INFO    MainThread:9152 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-08 00:41:51,011 INFO    MainThread:9152 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-03-08 00:41:51,011 INFO    MainThread:9152 [wandb_init.py:_log_setup():526] Logging user logs to C:\Users\hp\Desktop\CV\Assignment_2\wandb\run-20240308_004150-blaldr1o\logs\debug.log
2024-03-08 00:41:51,012 INFO    MainThread:9152 [wandb_init.py:_log_setup():527] Logging internal logs to C:\Users\hp\Desktop\CV\Assignment_2\wandb\run-20240308_004150-blaldr1o\logs\debug-internal.log
2024-03-08 00:41:51,012 INFO    MainThread:9152 [wandb_init.py:_jupyter_setup():472] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001C18F27FC90>
2024-03-08 00:41:51,012 INFO    MainThread:9152 [wandb_init.py:init():566] calling init triggers
2024-03-08 00:41:51,012 INFO    MainThread:9152 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'learning_rate': 0.001, 'batch_size': 60, 'optimizer': 'adam', '#Train Samples': 600}
2024-03-08 00:41:51,012 INFO    MainThread:9152 [wandb_init.py:init():591] re-initializing run, found existing run on stack: yb8ge5dy
2024-03-08 00:41:51,013 INFO    MainThread:9152 [wandb_run.py:_finish():1970] finishing run tejas591995/my-mnist-training/yb8ge5dy
2024-03-08 00:41:51,013 INFO    MainThread:9152 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-03-08 00:41:51,014 INFO    MainThread:9152 [wandb_run.py:_restore():2193] restore
2024-03-08 00:41:51,014 INFO    MainThread:9152 [wandb_run.py:_restore():2199] restore done
2024-03-08 00:41:57,152 INFO    MainThread:9152 [wandb_run.py:_footer_history_summary_info():3851] rendering history
2024-03-08 00:41:57,152 INFO    MainThread:9152 [wandb_run.py:_footer_history_summary_info():3883] rendering summary
2024-03-08 00:41:57,157 INFO    MainThread:9152 [wandb_run.py:_footer_sync_info():3810] logging synced files
2024-03-08 00:41:57,289 INFO    MainThread:9152 [wandb_init.py:init():616] starting backend
2024-03-08 00:41:57,289 INFO    MainThread:9152 [wandb_init.py:init():620] setting up manager
2024-03-08 00:41:57,291 INFO    MainThread:9152 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-03-08 00:41:57,292 INFO    MainThread:9152 [wandb_init.py:init():628] backend started and connected
2024-03-08 00:41:57,299 INFO    MainThread:9152 [wandb_run.py:_label_probe_notebook():1295] probe notebook
2024-03-08 00:41:57,302 INFO    MainThread:9152 [wandb_run.py:_label_probe_notebook():1305] Unable to probe notebook: 'charmap' codec can't decode byte 0x81 in position 265904: character maps to <undefined>
2024-03-08 00:41:57,302 INFO    MainThread:9152 [wandb_init.py:init():720] updated telemetry
2024-03-08 00:41:57,304 INFO    MainThread:9152 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-03-08 00:41:59,018 INFO    MainThread:9152 [wandb_run.py:_on_init():2262] communicating current version
2024-03-08 00:41:59,366 INFO    MainThread:9152 [wandb_run.py:_on_init():2271] got version response 
2024-03-08 00:41:59,366 INFO    MainThread:9152 [wandb_init.py:init():804] starting run threads in backend
2024-03-08 00:42:10,903 INFO    MainThread:9152 [wandb_run.py:_console_start():2241] atexit reg
2024-03-08 00:42:10,903 INFO    MainThread:9152 [wandb_run.py:_redirect():2096] redirect: wrap_raw
2024-03-08 00:42:10,903 INFO    MainThread:9152 [wandb_run.py:_redirect():2161] Wrapping output streams.
2024-03-08 00:42:10,903 INFO    MainThread:9152 [wandb_run.py:_redirect():2186] Redirects installed.
2024-03-08 00:42:10,903 INFO    MainThread:9152 [wandb_init.py:init():847] run started, returning control to user process
2024-03-08 00:42:33,433 INFO    MainThread:9152 [wandb_setup.py:_flush():76] Current SDK version is 0.16.4
2024-03-08 00:42:33,433 INFO    MainThread:9152 [wandb_setup.py:_flush():76] Configure stats pid to 9152
2024-03-08 00:42:33,433 INFO    MainThread:9152 [wandb_setup.py:_flush():76] Loading settings from C:\Users\hp\.config\wandb\settings
2024-03-08 00:42:33,434 INFO    MainThread:9152 [wandb_setup.py:_flush():76] Loading settings from C:\Users\hp\Desktop\CV\Assignment_2\wandb\settings
2024-03-08 00:42:33,434 INFO    MainThread:9152 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-03-08 00:42:33,434 INFO    MainThread:9152 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-08 00:42:33,434 INFO    MainThread:9152 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-03-08 00:42:33,434 INFO    MainThread:9152 [wandb_init.py:_log_setup():526] Logging user logs to C:\Users\hp\Desktop\CV\Assignment_2\wandb\run-20240308_004233-8ocwn63m\logs\debug.log
2024-03-08 00:42:33,434 INFO    MainThread:9152 [wandb_init.py:_log_setup():527] Logging internal logs to C:\Users\hp\Desktop\CV\Assignment_2\wandb\run-20240308_004233-8ocwn63m\logs\debug-internal.log
2024-03-08 00:42:33,434 INFO    MainThread:9152 [wandb_init.py:init():566] calling init triggers
2024-03-08 00:42:33,434 INFO    MainThread:9152 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'learning_rate': 0.001, 'batch_size': 60, 'optimizer': 'adam', '#Train Samples': 600}
2024-03-08 00:42:33,434 INFO    MainThread:9152 [wandb_init.py:init():591] re-initializing run, found existing run on stack: blaldr1o
2024-03-08 00:42:33,436 INFO    MainThread:9152 [wandb_run.py:_finish():1970] finishing run tejas591995/my-mnist-training/blaldr1o
2024-03-08 00:42:33,436 INFO    MainThread:9152 [jupyter.py:save_history():445] not saving jupyter history
2024-03-08 00:42:33,436 INFO    MainThread:9152 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-03-08 00:42:33,436 INFO    MainThread:9152 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
2024-03-08 00:42:33,436 INFO    MainThread:9152 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-03-08 00:42:33,436 INFO    MainThread:9152 [wandb_run.py:_restore():2193] restore
2024-03-08 00:42:33,437 INFO    MainThread:9152 [wandb_run.py:_restore():2199] restore done
2024-03-08 00:42:40,132 INFO    MainThread:9152 [wandb_run.py:_footer_history_summary_info():3851] rendering history
2024-03-08 00:42:40,132 INFO    MainThread:9152 [wandb_run.py:_footer_history_summary_info():3883] rendering summary
2024-03-08 00:42:40,136 INFO    MainThread:9152 [wandb_run.py:_footer_sync_info():3810] logging synced files
